docs/contributing/add_new_resource.md

```diff
@@ -99,3 +105,23 @@ Be sure to add test cases to [test/cli_test.go](https://github.com/GoogleCloudPl
    - A .json file (representing the output of `terraform-validator convert`)
 
 See [Getting started](./getting_started.md) for details on running tests.
+
+Try to get your tests passing locally before proceeding. (But you can also go ahead and open PRs if you're running into issues you can't figure out how to resolve.)
+
+### 3. Make PRs
+
+Now that you have your code working locally, open PRs for [Magic Modules](https://github.com/GoogleCloudPlatform/magic-modules) and terraform-validator. The reviewer will make sure your code works as expected.
+
+For the Magic Modules PR, the most important check is `terraform-google-conversion-test` - as long as that's passing, you're probably fine. If it is failing, go back to step 1 and try running `make test` for terraform-google-conversion to reproduce & fix the failure.
+
+For terraform-validator, the tests will not pass at this point, because the terraform-google-conversion dependency has not yet been updated. As long as the tests are passing locally for you, it should be fine.
+
+### 4. Update terraform-google-conversion dependency
+
+In your terraform-validator PR, update the terraform-google-conversion dependency. This command will make the necessary changes:
```

Comment:
"After the Magic Modules PR has merged, and the terraform-google-conversion repository contains your changes, ..." ?